java-numbers/src/main/java/com/baeldung/numberofdigits/NumberOfDigitsDriver.java

```diff
@@ -1,35 +1,36 @@
 package com.baeldung.numberofdigits;
 
-import org.apache.log4j.Logger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class NumberOfDigitsDriver {
     private static NumberOfDigits numberOfDigits;
-    
-    private static Logger LOG = Logger.getLogger(NumberOfDigitsDriver.class);
-    
+

```

Comment:
Hmm, while I agree that `slf4j` is totally the way to go, I think this task should just be about moving the classes over. Would you be okay with moving this back to `log4j` and logging an enhancement?